src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  margin-top: 20px;
+  width: 100%;
+  height: 70px;
+  box-sizing: border-box;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.form--small {
+  height: 42px;
+}
+
+.search {
+  position: relative;
+  top: 24px;
+  left: 25px;
+}
+
+.search--small {
+  top: 10px;
+  left: 11px;
+}
+
+.input {
+  position: relative;
+  bottom: 4px;
+  margin-left: 14px;
+  border-style: none;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.input::placeholder {
+  color: #3d4e61;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form:focus-within {
+  background-image: linear-gradient(180deg, #fff 0%,
+  #f6f6f7 100%);
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  border-radius: 3px;
+}
+
+.input:focus {
+  outline: none;
+  color: #3d4e61;
+  background-image: linear-gradient(180deg, #fff 0%,
+  #f6f6f7 100%);
+  font-weight: 800;
+}
+
+.input--small {
+  bottom: 1px;
+  margin-left: 7px;
```

Comment:
This values are strange. All values must be divisible by 4, if you have to adjust pixel by pixel for tests, there are some problems with centering or positioning, please check this